Fix scope of 'stack' when building with clang 7:

error: reference to 'stack' is ambiguous

--- src/FDM/UIUCModel/uiuc_aircraft.h.orig	2018-08-31 19:26:14.641298000 +0200
+++ src/FDM/UIUCModel/uiuc_aircraft.h	2018-08-31 19:50:35.444961000 +0200
@@ -150,7 +150,7 @@
 #include "uiuc_parsefile.h"
 #include "uiuc_flapdata.h"
 
-typedef stack :: iterator LIST;
+typedef ::stack :: iterator LIST;
 
 /* Add more keywords here if required*/
 enum {init_flag = 1000, geometry_flag, controlSurface_flag, controlsMixer_flag, 
--- src/FDM/UIUCModel/uiuc_parsefile.h.orig	2018-08-31 19:34:45.641135000 +0200
+++ src/FDM/UIUCModel/uiuc_parsefile.h	2018-08-31 19:35:35.903930000 +0200
@@ -23,7 +23,7 @@
 {
         private:
                 
-                stack commands;
+                ::stack commands;
                 ifstream file;
                 void readFile();
 
@@ -37,7 +37,7 @@
                 void removeComments(string& inputLine);
                 string getToken(string inputLine, int tokenNo);
                 void storeCommands(string inputLine);
-                stack getCommands();
+                ::stack getCommands();
 };
 
 #endif  // _PARSE_FILE_H_
--- src/FDM/UIUCModel/uiuc_recorder.cpp.orig	2018-08-31 19:16:32.680030000 +0200
+++ src/FDM/UIUCModel/uiuc_recorder.cpp	2018-08-31 19:16:57.872109000 +0200
@@ -102,7 +102,7 @@
 
 void uiuc_recorder( double dt )
 {
-  stack command_list;
+  ::stack command_list;
   string linetoken;
   // static int init = 0;
   static int recordStep = 0;
